src/Pointer.js

```diff
@@ -175,6 +194,10 @@ export class Pointer {
     this.removeEvent();
     const element = this.config.root || window;
     element.addEventListener('pointermove', this._measure, {passive: true});
+    if (this.config.scenes[0].allowActiveEvent) {
```

Comment:
This should be in the config of the timeline, not a scene
```suggestion
    if (this.config.allowActiveEvent) {
```

Reply:
Not sure I understand. Config of the timeline you mean we should get it from getCanvasCustomEffect?